File: src/main/java/br/com/controle/financeiro/model/repository/InstitutionRepository.java
package br.com.controle.financeiro.model.repository;

import java.util.UUID;

import br.com.controle.financeiro.model.entity.Institution;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface InstitutionRepository extends JpaRepository<Institution, UUID> {

}

File: src/main/kotlin/br/com/controle/financeiro/model/repository/InstitutionRepository.kt
package br.com.controle.financeiro.model.repository

import br.com.controle.financeiro.model.entity.Institution
import java.util.UUID
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.rest.core.annotation.RepositoryRestResource

@RepositoryRestResource interface InstitutionRepository : JpaRepository<Institution, UUID>
